pandas/tests/reshape/concat/test_datetimes.py

```diff
@@ -274,6 +274,42 @@ def test_concat_NaT_series_dataframe_all_NaT(self, tz1, tz2):
         result = concat([first, second])
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("unit", ["ns"])
+    def test_concat_series_columns_nonoverlap_5min_units(self, unit):
+        # GH#58471 minimal reproducible example
```

Comment:
nit: `minimal reproducible example` doesn't really add any information here and can be removed; it's generally applicable to all tests.